pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -20,12 +20,12 @@ limitations under the License.
 package cm
 
 import (
-	"bytes"
 	"context"
 	"fmt"
+	"k8s.io/kubernetes/pkg/kubelet/events"
```

Comment:
these 2 lines should be in the correct section (see the list of imports from line 39 through 68). these 2 lines belong there.